app/views/configure/reading.phtml

```diff
@@ -84,6 +84,44 @@
 				</div>
 			</div>
 
+			<div class="form-group">
+				<label class="group-name" for="secondary_sort_order"><?= _t('conf.reading.secondary_sort_order') ?></label>
+				<div class="group-controls">
+					<select name="secondary_sort_order" id="secondary_sort_order">
+						<option value="DESC"<?= FreshRSS_Context::userConf()->secondary_sort_order === 'DESC' ? ' selected="selected"' : '' ?>><?= _t('conf.reading.secondary_sort_order.desc') ?></option>
+						<option value="ASC"<?= FreshRSS_Context::userConf()->secondary_sort_order === 'ASC' ? ' selected="selected"' : '' ?>><?= _t('conf.reading.secondary_sort_order.asc') ?></option>
+					</select>
+				</div>
+			</div>
+
+			<?php $secondary_sort = FreshRSS_Context::userConf()->secondary_sort?>
+			<div class="form-group">
+			<label class="group-name" for="secondary_sort"><?= _t('conf.reading.secondary_sort') ?></label>
```

Comment:
We should probably add a configuration for *primary sort* as well. It would be strange to be able to change the secondary sort but not the primary one